docs/user_guides/templates/simulation_structures.mdx

```diff
@@ -35,16 +35,15 @@ to update the rendering of only the rigid-bodies that moved:
 <rapier>
 
 ```rust
-// Iter on each dynamic rigid-bodies that moved.
-for rigid_body_handle in island_manager.active_dynamic_bodies() {
-    let rigid_body = &rigid_body_set[*rigid_body_handle];
-    println!("Rigid body {:?} has a new position: {}", rigid_body_handle, rigid_body.position());
-}
+<load path='/2d/rust/examples/rs_simulation_structures2.rs' marker='IslandManager' />
 ```
 
 </rapier>
-<bevy>
 
+<bevy>
+<technote>
+this is not used, nothing links to it. Also, it's not compiling.
+</technote>
```

Comment:
I'm not sure how to make comments, `<!-- -->` results in a `yarn start` fail, so that's my workaround ; I can remove if unwelcome.